packages/desktop-client/src/components/budget/misc.js

```diff
@@ -352,12 +383,15 @@ export function SidebarCategory({
               onMenuSelect={type => {
                 if (type === 'rename') {
                   onEditName(category.id);
-                } else {
+                } else if (type === 'delete') {
                   onDelete(category.id);
+                } else if (type === 'toggleHide') {
+                  onSave({ ...category, hidden: !category.hidden });
                 }
                 setMenuOpen(false);
               }}
               items={[
+                { name: 'toggleHide', text: category.hidden ? 'Show' : 'Hide' },
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
                { name: 'toggleVisibility', text: category.hidden ? 'Show' : 'Hide' },
```